src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
-body {
+$color-block: #eee;
+$start-days: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+$month-lengths: (28, 29, 30, 31);
+$calendar-width: 706;
+
+.page {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
 }
+
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  cursor: pointer;
+  display: flex;
+  flex-flow: row wrap;
+  margin: 81.5px 151px;
+  gap: 1px;
+  width: $calendar-width;
+  justify-content: flex-start;
+  list-style: none;
+  padding: 0;
+
+  &__day {
+    width: 100px;
+    height: 100px;
+    text-align: center;
+    line-height: 98px;
```

Comment:
The `line-height` here is almost equal to the height of the element, which is `100px`. However, since you have `border: 1px solid black;` on line 40, the actual content height is `98px` (100px - 2px for top and bottom borders). To vertically center the text accurately, `line-height` should also be `98px`.